tests/integration-tests/src/flows/faucet/initFaucet.ts

```diff
@@ -0,0 +1,45 @@
+import { FlowProps } from '../../Flow'
+import { extendDebug } from '../../Debugger'
+import { FixtureRunner } from '../../Fixture'
+import { BuyMembershipHappyCaseFixture, TransferInvitesHappyCaseFixture } from '../../fixtures/membership'
+import { SetBudgetFixture } from '../../fixtures/workingGroups/SetBudgetFixture'
+import _ from 'lodash'
+import BN from 'bn.js'
+
+export default async function initFaucet({ api, env, query }: FlowProps): Promise<void> {
+  const debug = extendDebug('flow:initFaucet')
+  debug('Started')
+
+  // Get membership working group leader
+  const membershipWorkingGroup = 'membershipWorkingGroup'
+  const [, membershipLeader] = await api.getLeader(membershipWorkingGroup)
+
+  // Grant lead invitation quota
+  const proposalDetails = api.createType('ProposalDetails', { SetMembershipLeadInvitationQuota: 50 })
+
+  // The membership working group should have a budget allocated
+  const budgets: BN[] = [new BN(1000000)]
+  const setGroupBudgetFixture = new SetBudgetFixture(api, query, membershipWorkingGroup, budgets)
+  await new FixtureRunner(setGroupBudgetFixture).runWithQueryNodeChecks()
+
+  // Create a membership account for faucet
+  const faucetAccounts = (await api.createKeyPairs(1)).map(({ key }) => key.address)
```

Comment:
we could also get the account from the env.

Reply:
Done